package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "openmrs-owa-platformui",
+  "version": "0.1.0",
+  "description": "A very simple UI for the OpenMRS Platform to give access to the Addon Manager, User manager, and other tools.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/charpell/openmrs-owa-openmrs-owa-platformui"
```

Comment:
Can we change this url to that of openmrs?

Reply:
I have updated the PR